src/scripts/main.js

```diff
@@ -354,7 +354,22 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that there is an element with the class 'dashboard' in your HTML. If this element does not exist, the `querySelector` will return `null`, and attempting to append children to `null` will result in a runtime error.